site/src/components/IconWrapper.tsx

```diff
@@ -17,11 +17,12 @@ export const IconWrapper = forwardRef<SVGSVGElement, IconWrapperProps>((props, r
     strokeLinejoin: 'round',
   };
 
-  const { content, ...rest } = props;
+  const { src, ...rest } = props;
   const attrs = {
     ...defaultAttrs,
     ...rest,
   };
+  const content = src.replace(/<svg[^>]*>|<\/svg>/g, '');
```

Comment:
Core change is here. We no longer need to pass `content`, which is essentially the same thing as `src` minus `<svg>` wrap.

Reply:
Nice findings!